### DISCUSSION TITLE
Dev Mode?

### DISCUSSION THREAD

**Discussion by JoeMheh** [3kh0/ext-remover#621] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE] [answered]
How would I get into Dev mode on the Chromebook without like unenrolling? Or do I have to unenroll?

**Comment by pazza2234** [comment, marked as answer]
Thats the neat part. you don't. well, with out the it guy comming for your blood. if you got a pc, and you want to stay enrolled your only choice is really fakmurk and sh1mmer. sh1mmer for dev mode, and fakmurk to hide from IT and let you boot from a USB stick. after you are done installing both find a way to make a persistent USB stick of a linux distro (I recommend puppy linux) remember I am telling you how and if you get caught no one but you will get in trouble. and (eventually) everyone will get caught. I will not give more info find a discord or something for help after that. also just don't do this. I am just answering your question.

**Comment by AnOrangeCone** [comment]
You can't without using fakemurk but thats still unenrolling
